dask/dataframe/core.py

```diff
@@ -333,12 +333,22 @@ def iloc(self):
         # see https://github.com/ContinuumIO/dask/pull/507
         raise AttributeError("Dask Dataframe does not support iloc")
 
-    def repartition(self, divisions):
+    def repartition(self, divisions, force=False):
         """ Repartition dataframe along new divisions
 
+        Parameters
+        ----------
+
+        divisions: list
+            List of partitions to be used
+        force: bool, default False
+            Whether to allow expand the existing divisions.
```

Comment:
Ah, I see that `expand` is correct and adding `correct` is incorrect.  My apologies.